crates/wash-lib/src/parser/mod.rs

```diff
@@ -116,7 +116,7 @@ impl TryFrom<RawActorConfig> for ActorConfig {
             // TODO(#1624): Default to ~/.wash/keys
             key_directory: raw_config
                 .key_directory
-                .unwrap_or_else(|| PathBuf::from("./keys")),
+                .unwrap_or_else(|| PathBuf::from("~/.wash/keys")),
```

Comment:
Just a note for later that `~` will probably work well on all unix systems but _might_ be strange on Windows. Last I checked it does work with an alias on Windows 11, but we may want to consider a crate like https://crates.io/crates/homedir (perhaps we already have one in `Cargo.toml` that I'm forgetting about?) for this

Reply:
my bad. I completely forgot to think of other OS. really appreciate the Rust crate link. let me research a little on that.

Reply:
hi @brooksmtownsend , agreed. I was thinking of identifying the issue on unix systems first and getting the keys generated just in the user's $HOME directory. I can then take up the task task for `wash build` to detect the user's OS. Please let me know your thoughts on this :)

Reply:
Heads up that we have the `dirs` crate available that can help with the home directory: https://docs.rs/dirs